webpack.config.ts

```diff
@@ -12,9 +12,6 @@ module.exports = {
           {
             loader: 'ts-loader',
           },
-          {
```

Comment:
Could you also remove this https://github.com/Redocly/redocly-cli/pull/863/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L68 ?

Reply:
Thanks, I missed that. removed.